oauth2-authorization-server/src/test/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2AuthorizationCodeGrantTests.java

```diff
@@ -389,6 +390,10 @@ OAuth2AuthorizationService authorizationService() {
 				}
 			};
 		}
+		@Bean
+		JwtDecoder jwtDecoder(){
```

Comment:
This should not be required. It's only required if `resourceServer()` is applied, which is only applied if the registration endpoint is enabled.

Please revert this change and to all integration tests.